www/js/data.js

```diff
@@ -20,7 +20,9 @@ elephant.constant('elephantData_URL', {
   GIVEN_AWAY_ITEM: 'http://'+getUrl(window.location.hostname)+'/givenaway.php',
   GIVEN_AWAY_TYPE: 'POST',
   RE_APPROVE_ITEM: 'http://'+getUrl(window.location.hostname)+'/reapprove.php',
```

Comment:
reapprove.php is chaging item status to 1 which means item approve and admin is the only person who the right to change status to 1. So in this case, if the a user triggers reapprove function, it will turn the item status to 1 without having the admin approval. However, if it is a different case, then I would require a bit more clarity on what is happening here.